tests/CMakeLists.txt

```diff
@@ -9,7 +9,8 @@ set(CARAMEL_TESTS
     BucketedHashStoreTest
     GaussianEliminationTest
     HypergraphPeelerTest
-    LazyGaussianEliminationTest)
+    LazyGaussianEliminationTest
+    SolveTest)
```

Comment:
did you mean to add a SolveTest somewhere?

Reply:
Oh thanks, I forgot to include this. 

Now that the .gitignore is fixed, it'll be harder to miss stuff like this.